go-controller/hack/test-go.sh

```diff
@@ -49,6 +49,10 @@ function testrun {
     if [ ! -z "${COVERALLS:-}" ]; then
         args="${args} -test.coverprofile=${idx}.coverprofile "
     fi
+    if [[ " ${big_pkgs[@]} " =~ " $pkg " ]]; then
```

Comment:
why not just set it 20 all the time?

Reply:
We could do that.

But I think it's pretty uncommon and undesirable to have package UTs running for that long. We should be looking into breaking ovn package down into smaller things rather than generally be increasing timeouts.

And if some other package UT hangs for any reason, we still only wait for 10 mins rather than for 20 or whatever we increase it to.